andela/krypton-ah-frontend
# [#163693759] - landing-page-logic-implementation

#### What does this PR do?
This PR integrates the components with the reducers and actions

#### Description of Task to be completed?
To link the login frontend to the backend

#### How should this be manually tested?
Git clone the repo and cd into it
npm run dev to start the server
open the redux dev tools to check the application state

#### What are the relevant pivotal tracker stories?
#163693759

##Screenshots
![landing-page-css-fix](https://user-images.githubusercontent.com/40843132/53654710-abdcca00-3c4e-11e9-969f-336b90e75269.gif)
